### DISCUSSION TITLE
Orchid login with LDAP

### DISCUSSION THREAD

**Discussion by elbaku** [orchidsoftware/platform#2088] [Q&A] [answered]
Hello guys, has anyone implemented a login with ldap using Orchid?

**Comment by elbaku** [comment, marked as answer]
After a few hours hitting keys on my keyboard i stumbled upon a library called [adldap2/adldap2-laravel](https://github.com/Adldap2/Adldap2-Laravel), integrating it was not that difficult. 

Thanks :D

**Reply by LazyGatto** [reply]
Actually, better solution for now is to use https://ldaprecord.com/docs/laravel/v2 ;)

**Reply by sebastianpospisil** [reply]
Hello @LazyGatto,
currently, I'm trying to get behind how to implement ldaprecord in orchid. I can't seem to find where I have to implement the login procedure. Do I have to follow the tutorial about Laravel UI on their website or one of the other ones (Laravel Breeze, Laravel Jetstream
Laravel Sanctum)? Also, what do I keep track of in the user table when installing their database auth?
Greetings Sebastian
